en-US/firefox/whatsnew_70.lang

```diff
@@ -0,0 +1,72 @@
+## NOTE: https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/70.0/whatsnew
+
+
+# HTML page title.
+;What’s new with Firefox - More privacy, more protections.
+What’s new with Firefox - More privacy, more protections.
+
+
+# Notification banner.
+;Congrats! You’re using the latest version of Firefox.
+Congrats! You’re using the latest version of Firefox.
+
+
+# Page headline.
+;Now you can see how we’re protecting your privacy.
+Now you can see how we’re protecting your privacy.
+
+
+;We made Enhanced Tracking Protection because no one should be able to follow you online without your permission. And now you can see just how many data-collecting trackers Firefox has blocked for you — along with other ways our products protect your privacy.
+We made Enhanced Tracking Protection because no one should be able to follow you online without your permission. And now you can see just how many data-collecting trackers Firefox has blocked for you — along with other ways our products protect your privacy.
+
+
+# Conditional CTA for Firefox 70 and above.
+;See what Firefox has blocked for you
+See what Firefox has blocked for you
+
+
+# Conditional CTA for Firefox 69 and below.
+;Update your Firefox browser
+Update your Firefox browser
+
+
+# Conditional CTA for non-Firefox browsers.
+;Download the Firefox browser
+Download the Firefox browser
+
+
+;Also view personalized protection updates from these services:
+Also view personalized protection updates from these services:
+
+
+# Monitor refers to the Firefox Monitor product. It should remain in English.
+;Firefox Monitor
+Firefox Monitor
+
+
+;See if you were a part of another company’s data breach, and sign up for future alerts.
+See if you were a part of another company’s data breach, and sign up for future alerts.
+
+
+# Monitor refers to the Firefox Monitor product. It should remain in English.
+;Go to Monitor
+Go to Monitor
+
+
+# Lockwise refers to the Firefox Lockwise product. It should remain in English.
```

Comment:
Nitpicking: let's change to `Firefox Lockwise should remain unchanged.` since the string contains the full name.